api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceTest.java

```diff
@@ -165,8 +169,9 @@ public class WorkspaceAdminServiceTest {
   })
   static class Configuration {
     @Bean
-    public WorkbenchConfig getConfig() {
-      return WorkbenchConfig.createEmptyConfig();
+    @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
```

Comment:
What does this do?

Reply:
This is required to retrieve the configuration feature flag value. Without this i was getting an empty configuration object or the value i am trying to set in test doesnt stick.  More info:  https://www.baeldung.com/spring-bean-scopes